ang/civicase/CaseCard.js

```diff
@@ -15,12 +15,22 @@
   });
 
   module.controller('CivicaseCaseCardController', function ($scope, getActivityFeedUrl, DateHelper) {
+    $scope.data.selected = $scope.data.selected || false;
+
     $scope.activityFeedUrl = getActivityFeedUrl;
     $scope.formatDate = DateHelper.formatDate;
     $scope.otherCategories = _.map(_.filter(CRM.civicase.activityCategories, function (category) {
       return category.name !== 'task' && category.name !== 'communication';
     }), function (category) {
       return category.name;
     });
+
+    /**
+     * Update the checkbox the case card
```

Comment:
There are two kings of checkbox in the case card, we need to be more specific.
`Update Bulk Action checkbox of the case card`